src/applications/personalization/dashboard/components/claims-and-appeals/Appeal.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import moment from 'moment';
 
 import {
```

Comment:
similar to Adam's comment above. should we rename `appeals-v2-helpers.jsx` to just `appeals-helpers.jsx?`